pkg/apis/config/types.go

```diff
@@ -73,3 +74,17 @@ type CapacitySchedulingArgs struct {
 	// KubeConfigPath is the path of kubeconfig.
 	KubeConfigPath string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// TargetLoadPackingArgs holds arguments used to configure TargetLoadPacking plugin.
+type TargetLoadPackingArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Node target CPU Utilisation for bin packing
+	TargetCPUUtilization *float64 `json:"targetCPUUtilization,omitempty"`
```

Comment:
> (BTW: mentioned above, hardcoded TargetCPUUtilization name doesn't sound like a good idea)

Not able to find if you are referring to another comment or some point in best practices. Looked elsewhere, only `DefaultRequests` was suggested instead of `DefaultCPURequests`. Did you imply this should be `TargetUtilization?`

Reply:
Updated to `TargetUtilization`